graphql/graphql.go

```diff
@@ -1207,6 +1207,9 @@ func (r *Resolver) Block(ctx context.Context, args struct {
 	Number *Long
 	Hash   *common.Hash
 }) (*Block, error) {
+	if args.Number != nil && args.Hash != nil {
+		return nil, errors.New("either number or hash must be specified")
```

Comment:
I don't think it's really right. It reads as if the error is that nothing has been specified. 
How about something more like these? I'm not super-happy about them either.. :/ 

```suggestion
		return nil, errors.New("only one of [number,hash] must be specified")
```
```suggestion
		return nil, errors.New("specify either number nor hash, not both")
```

Reply:
I changed to `only one of number or hash must be specified`. Hopefully that's more clear